Cargo.toml

```diff
@@ -67,3 +67,5 @@ name = "r1cs"
 harness = false
 required-features = ["yoloproofs", "avx2_backend"]
 
+[patch.crates-io]
+curve25519-dalek = { git = "https://github.com/mobilecoinfoundation/curve25519-dalek.git", rev = "8791722e0273762552c9a056eaccb7df6baf44d7" }
```

Comment:
This can be removed when https://github.com/dalek-cryptography/curve25519-dalek/pull/383 is published